src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorGenerateWithTargetPath.cs

```diff
@@ -228,7 +231,8 @@ private void InitializeFallbackRule()
     }
     
     internal partial class RazorProjectProperties
-    {  
+    {
+        
```

Comment:
I hate the extra spaces this file generates.

Reply:
This should be fixed on master now